packages/block-editor/src/components/grid/style.scss

```diff
@@ -208,5 +242,21 @@
 		}
 	}
 
+	.block-editor-grid-item-mover-button {
+		white-space: nowrap;
```

Comment:
To prevent text wrapping when the "Show button text labels" is enabled

![image](https://github.com/user-attachments/assets/032ce202-306b-4751-b806-b11115b897b5)